validator/testdata/feature_tests/amp-embedly-card.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
```

Comment:
This should probably be in `extensions/amp-embedly-card/0.1/tests/` as `validator-amp-embedly-card.html` and the `.out` file as well.

Reply:
Moved to recommended folder and renamed.